src/scss/tracks.scss

```diff
@@ -208,9 +208,12 @@
 	}
 }
 
-.tracks.blurred,
+.tracks.blurred {
```

Comment:
These could be combined into the combined rule as they were originally, shouldn't need the repitition. 

```css
.tracks.blurred,
.enactedpolicies-container.blurred {
	filter: saturate(60%);
	opacity: 0.3;
}
```

Reply:
Got it